src/main/java/lt/vu/web/api/v1/dto/order/PutOrderStatusDTO.java

```diff
@@ -0,0 +1,15 @@
+package lt.vu.web.api.v1.dto.order;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.FormParam;
+
+@Getter @Setter
+public class PutOrderStatusDTO {
```

Comment:
This is redundant, id should come from path param (`api/v1/orders/{id}/change-status}`). This dto could in turn accept either `DELIVERED`, either `NEW` (just so it's future-proof in case we invent new statuses).